hotness/tests/test_consumer.py

```diff
@@ -0,0 +1,130 @@
+"""
+Unit tests for hotness.consumer
+"""
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+
+import mock
+
+import fedmsg.config
```

Comment:
This should be grouped with 3rd-party libraries (mock)

Reply:
Agreed.  Fixed in 79d4ec3.